diff -ruN /mnt/data/base155src/mesh-messenger-v1.5.5-work/android-app/app/src/main/assets/www/sw.js /mnt/data/build_alpha1/android-app/app/src/main/assets/www/sw.js
--- /mnt/data/base155src/mesh-messenger-v1.5.5-work/android-app/app/src/main/assets/www/sw.js	2026-09-21 17:41:57.000000000 +0000
+++ /mnt/data/build_alpha1/android-app/app/src/main/assets/www/sw.js	2026-09-21 21:36:15.000000000 +0000
@@ -1,4 +1,4 @@
-const CACHE = 'mesh-messenger-v155';
+const CACHE = 'mesh-messenger-v160-alpha1';
 const ASSETS = [
   './',
   './index.html',
@@ -14,6 +14,8 @@
   './src/transports/phoneapi-session.js',
   './src/transports/protobuf-lite.js',
   './src/transports/radio-transport.js',
+  './src/transports/lan-peer-transport.js',
+  './src/transports/internet-relay-transport.js',
   './src/transports/web-serial-meshtastic.js',
 ];
 self.addEventListener('install', event => event.waitUntil(
